OpenRA.Game/MiniYaml.cs

```diff
@@ -313,10 +315,12 @@ public static List<MiniYamlNode> FromString(string text, string fileName = "<no
 
 		public static List<MiniYamlNode> Merge(IEnumerable<List<MiniYamlNode>> sources)
 		{
+			sources = sources.ToList();
 			if (!sources.Any())
```

Comment:
Since this is a list now, can't we use `.Count == 0`?

Reply:
Updated